pkg/authorization/maximal_permission_policy_authorizer.go

```diff
@@ -140,30 +116,15 @@ func (a *MaximalPermissionPolicyAuthorizer) Authorize(ctx context.Context, attr
 
 	// If we can't find the export default to close
 	if !found {
-		kaudit.AddAuditAnnotations(
-			ctx,
-			MaximalPermissionPolicyAuditDecision, DecisionNoOpinion,
-			MaximalPermissionPolicyAuditReason, fmt.Sprintf("API export %q not found, path: %q", exportName, path),
-		)
-		return authorizer.DecisionNoOpinion, MaximalPermissionPolicyAccessNotPermittedReason, err
+		return authorizer.DecisionNoOpinion, MaximalPermissionPolicyAccessNotPermittedReason, fmt.Errorf("API export %q not found, path: %q", exportName, path)
```

Comment:
this is strange. Why not `authorizer.DecisionNoOpinion, fmt.Errorf("API export %q not found, path: %q", exportName, path), err` ?

Reply:
fixed

Reply:
side note: the err was a bug, there is no error present in this case, hence changed to `return authorizer.DecisionNoOpinion, fmt.Sprintf("API export %q not found, path: %q", exportName, path), nil`